File: SDKs/googleapis.com/artifactregistry/v1/go/pkg/models/shared/importyumartifactsgcssource.go
package shared

// ImportYumArtifactsGcsSource
// Google Cloud Storage location where the artifacts currently reside.
type ImportYumArtifactsGcsSource struct {
	Uris         []string `json:"uris,omitempty"`
	UseWildcards *bool    `json:"useWildcards,omitempty"`
}

File: SDKs/googleapis.com/artifactregistry/v1/typescript/src/sdk/models/shared/importyumartifactsgcssource.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// ImportYumArtifactsGcsSource
/** 
 * Google Cloud Storage location where the artifacts currently reside.
**/
export class ImportYumArtifactsGcsSource extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=uris" })
  uris?: string[];

  @SpeakeasyMetadata({ data: "json, name=useWildcards" })
  useWildcards?: boolean;
}
